app/jobs/nb_person_push_job.rb

```diff
@@ -2,7 +2,12 @@ class NbPersonPushJob < ActiveJob::Base
   queue_as :default
 
   def perform(person_attributes)
+    event_id = person_attributes.delete(:event_id)
     nb_args = Integration::NationBuilder.person_params(person_attributes)
-    Integration::NationBuilder.create_or_update_person(nb_args)
+    response = Integration::NationBuilder.create_or_update_person(nb_args) || {}
+    person_id = response['id']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.